community/matrix.md

```diff
@@ -8,33 +8,42 @@ editor: markdown
 dateCreated: 2022-01-26T00:49:51.001Z
 ---
 
-We have a Matrix homeserver at `matrix.mathesar.org`, but you can join our public rooms using any Matrix account. If you haven't used Matrix before, we recommend getting started [using Element](https://element.io/get-started).
+For realtime chat, we use [Matrix](https://matrix.org/) *(similar to Slack, Discord, Mattermost, Zulip, etc)*.
 
-A list of our public rooms is below. You can also join our [Matrix Space](https://matrix.to/#/!KQLkDbtIMsvcwUIfNy:matrix.mathesar.org?via=matrix.mathesar.org&via=matrix.org) which contains all these rooms.
+## Joining
 
-## General discussion
+1. Join our [Matrix Space](https://matrix.to/#/!KQLkDbtIMsvcwUIfNy:matrix.mathesar.org?via=matrix.mathesar.org&via=matrix.org).
```

Comment:
Joining our space is an optional thing, it's for convenience if they want to see all our rooms in one place in their Matrix client.

Reply:
I've merged this content, but I didn't address this point. I'm still a bit confused about it. I haven't taken the time to understand Matrix well enough to understand what "joining a space" means -- and how I would chat with our team _without_ joining our space. My thinking here is that this "Matrix Space" link is probably the quickest path towards getting a Matrix-novice such as me to chat with our team. The fewer steps and decisions, the better. And if someone _does_ understand more about Matrix they can probably figure out how to chat with us without joining our space. Am I correct here? If not, then would you be willing to supply the documentation content that instructs people on: (1) how to make a decision about whether or not to join the space, and (2) how to chat with us without joining the space?

Reply:
Spaces are essentially groups of Matrix channels. For example, in the Slack sidebar, you can organize all the Slack channels you've joined. Here's a screenshot of my "Social" group on the CCI Slack:

![Screen Shot 2022-01-27 at 4 21 46 PM](https://user-images.githubusercontent.com/287034/151445778-2cf178cc-8e2c-4af5-88df-ed26c5cc2384.png)

Now imagine that I can share this organization with other people, so that others on the CCI Slack can also have their channels sorted into the same categories as me. So when someone joins `#food`, it would show up under the Social category in the sidebar for them. That's equivalent to a Matrix Space. A different way to think about it would be as a namespace.

The Mathesar Matrix Space is basically that, it groups all the Mathesar channels together. So if someone joined the space, it would be an organizational aid to them. But it doesn't have anything to do with whether they join individual channels. It's probably not very useful if they're only joining one channel or if they're only using Matrix for Mathesar related chat and not joining any non-Mathesar related Matrix channels.

I hope that helps. [Here's a blog post about Spaces](https://element.io/blog/spaces-the-next-frontier/) also.

Reply:
Here are my edits: https://github.com/centerofci/mathesar-wiki/commit/58e43d59e0acae15a698b4de6c5a033c1a19357c#diff-4dae2430228d88e8482d74b816f47b8a5340f421bc4a087517d40ec8fb053e51 (I needed to re-sync the wiki, so they got added as part of a massive commit)

Reply:
Your edits look great. Thanks @kgodey